README.md

```diff
@@ -149,7 +149,6 @@ Flags:
   -h, --help                         help for baton-azure-infrastructure
       --log-format string            The output format for logs: json, console ($BATON_LOG_FORMAT) (default "json")
       --log-level string             The log level: debug, info, warn, error ($BATON_LOG_LEVEL) (default "info")
-      --mailboxSettings              If true, attempt to get mailbox settings for users to determine user purpose ($BATON_MAILBOXSETTINGS)
```

Comment:
@sergiocorral-conductorone I think we do not want to break connectors for existing customers